### ISSUE TITLE
fs_game write protected

### ISSUE THREAD

**Issue by Yetta1** [RobertBeckebans/RBDOOM-3-BFG#238]
Tried loading custom content through fs_game, gave the write protected message. Not a big deal, just wanted to try load old maps and mods from vanilla doom 3.

**Comment by BielBdeLuna** [comment]
what? fs_game is a cvar, how can it be write protected?

**Comment by Yetta1** [comment]
I don't know, the console just says it is write protected.

**Comment by BielBdeLuna** [comment]
oh, now I see idTechX, doesn't have "reloadEngine", I guess that's due how the engine manages the assets, (the fact that there isn't a game library is also an evidence of this), so you can only set the fs_game when you launch the engine and not within the engine, I guess that is one of the changes since idTech4.

so you want check a mod? you exit the game and relaunch the game with "+set fs_game yourmodfolder"

also I use this alongside that:

```
./idtechx_engine_of_your_preference +set fs_resourceLoadPriority 0 +set fs_copyfiles 0 +set image_highQualityCompression 0 +set r_useOpenGL32 0 +set fs_game your_mod_folder
```

**Comment by Yetta1** [comment]
I have tried +set fs_game with a shortcut, it reverted to D3BFG. Tried adding the line you provided same thing. Tried manually typing those in the console, fs_copyfiles also give write protected message.

**Comment by BielBdeLuna** [comment]
no, you have to add this line after the calling of the executable, in windows you could use a bat file or the way however you enter info in a CLUI style.

**Comment by Yetta1** [comment]
I've tried a bat file, doesn't seem to work. Still reverts to D3bfg screen. Tried manually opening a map, won't even recognise the directory.

I think there may also be a clash of the old Doom 3 gui with the new, as some of those mods used the old system.

I tried replacing the base folder with the mod folder and kept files needed from base, there was some luck loading some of the mod on OTE, but failures in some scripts which stopped map loading.

We can focus on this issue at a later stage in development to find a workaround for "legacy" D3 mods.
